xml/FrameworksIndex/windowsdesktop-3.0.xml

```diff
@@ -53238,6 +53238,7 @@
       <Member Id="M:System.Windows.Media.Animation.Storyboard.Stop(System.Windows.FrameworkElement)" />
       <Member Id="P:System.Windows.Media.Animation.Storyboard.Target" />
       <Member Id="P:System.Windows.Media.Animation.Storyboard.TargetName" />
+      <Member Id="P:System.Windows.Media.Animation.Storyboard.TargetProperty" />
```

Comment:
I think all of these end up conflicting with https://docs.microsoft.com/en-us/dotnet/api/system.windows.media.animation.storyboard.targetproperty?view=windowsdesktop-5.0

This may be why they weren't here, I feel like they were removed at one point.

Reply:
That's what this version of mdoc fixes. They've differentiated the URLs, so the attached properties now have "_a" on the end.